base/assets/minecraft/atlases/blocks.json

```diff
@@ -0,0 +1,14 @@
+{
+    "sources": [
+        {
+            "type": "directory",
+            "source": "gui",
+            "prefix": "gui/"
+        },
+        {
+            "type": "directory",
+            "source": "entity",
+            "prefix": "entity/"
```

Comment:
Can you elaborate on what you mean by that? Don't these configure additional atlases to create?

Reply:
The `blocks` atlas is a single texture atlas that is use by block and item model rendering (ignoring special entity renderers). So adding more texture to this makes the atlas larger. If the textures aren't used anywhere in item or block models they're just wasted space.

Reply:
I believe since all our textures are rendered by item model (on a display entity or actual item) they would need to be in this atlas then?

The `gui` directory is intended for things that show up in menus, so has stuff for guidebook and fonts. Do they need to get their own atlas, or would they not need an atlas at all?